docroot/profiles/custom/cgov_site/modules/custom/cgov_infographic/src/Controller/CGovInfographicController.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Drupal\cgov_infographic\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+
+/**
+ * Controller routines for cgov_infographic routes.
+ */
+class CGovInfographicController extends ControllerBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function getModuleName() {
+    return 'cgov_infographic';
+  }
+
+  /**
+   * Return an infographic's long description in an HTTP Response.
+   *
+   * @param int $node
+   *   Id of the infographic to render.
+   */
+  public function longDescription($node) {
+
+    // Check for invalid inputs.
+    if (!is_numeric($node)) {
+      throw new NotFoundHttpException();
+    }
+
+    // Load the entity, and verify that it's an infographic.
+    $infographic = $this->entityTypeManager()->getStorage('media')->load($node);
```

Comment:
if you use the "upcasting" configuration as defined in https://www.drupal.org/docs/8/api/routing-system/parameter-upcasting-in-routes and referenced in my comment on the routing.yml file, then this is not needed.  Once again, `getStorage('media')->load($node)` should never be written again, ever, never ever ever.

Reply:
Lots of finicky bits, but that's in place now.